libexec/bats-core/bats

```diff
@@ -57,7 +59,17 @@ arguments=()
 for arg in "$@"; do
   if [[ "${arg:0:1}" = "-" ]]; then
     if [[ "${arg:1:1}" = "-" ]]; then
-      options[${#options[*]}]="${arg:2}"
+      if [ "$(expr "${arg:2}" : "global-helper=.*$")" -ne 0 ]; then
+        BATS_GLOBAL_HELPER_SCRIPT="${arg##*=}"
+
+        if [ -z "${BATS_GLOBAL_HELPER_SCRIPT}" ]; then
```

Comment:
Use `[[ ]]` instead. Also, the `export BATS_GLOBAL_HELPER_SCRIPT` need not be in an `else` clause. In fact, it should probably go at the top of the file.